src/material/badge/badge.ts

```diff
@@ -156,6 +156,20 @@ export class MatBadge extends _MatBadgeBase implements OnInit, OnDestroy, CanDis
       if (nativeElement.nodeType !== nativeElement.ELEMENT_NODE) {
         throw Error('matBadge must be attached to an element node.');
       }
+
+      const badgeTagName: string = 'mat-icon';
```

Comment:
Isn't this the icon tag name?

Reply:
yes. I forget to make this change after copying content from my previous PR. I've fixed this in latest commit.